src/styles/styles.js

```diff
@@ -2693,6 +2693,17 @@ const styles = {
         maxWidth: 450,
         alignSelf: 'center',
     },
+
+    textPill: {
+        ellipsizeMode: 'end',
```

Comment:
Can you attach screenshots with the ellipsizeMode? Say a longer workspace name?

Reply:
Done. Updated the screenshots.